x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/CharSeqTokenTrieNode.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.ml.inference.nlp.tokenizers;
+
+import org.apache.lucene.analysis.CharArrayMap;
+import org.elasticsearch.core.CheckedFunction;
+import org.elasticsearch.core.Nullable;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+public class CharSeqTokenTrieNode {
```

Comment:
This doesn't look like it's a move from `TokenTrieNode.java`. Do we still have that file? Should it be deleted?

Reply:
`TokenTrieNode` should be deleted. git might not have treated it like a move. I will look.